dali/test/python/test_fw_iterators.py

```diff
@@ -1728,7 +1728,9 @@ def BoringPipeline():
 
     loader = fw_iterator(pipeline, reader_name="reader", auto_reset=auto_reset_op)
     for _ in range(2):
-        for i, data in enumerate(loader):
+        loader = iter(loader)
```

Comment:
Hmmm... at this point I'd suggest to store it into a separate variable, should we switch to returning something else than `self` in the future.

Reply:
Fixed